executor/aggregate.go

```diff
@@ -191,9 +193,18 @@ type HashAggExec struct {
 	prepared                bool
 	executed                bool
 
-	memTracker *memory.Tracker // track memory usage.
+	memTracker  *memory.Tracker // track memory usage.
+	diskTracker *disk.Tracker
 
 	stats *HashAggRuntimeStats
+
+	listInDisk   *chunk.ListInDisk   // listInDisk is the chunks to store row values for spilling data.
+	lastChunkNum int                 // lastChunkNum indicates the num of spilling chunk.
+	processIdx   int                 // processIdx indicates the num of processed chunk in disk.
```

Comment:
offsetOfSpillChks

Reply:
This comment is hard to understand, we need a detailed comment to explain the usage of this var

Reply:
Addressed.